app/controllers/users_controller.rb

```diff
@@ -69,28 +69,20 @@ def destroy
   private
```

Comment:
- Mirate 1º el description =)
- No se si la idea era mas o menos esta o en este caso que son todos like's preferís que hagamos un split (por AND/OR) y armar las cosas medio a mano (que sería parte de lo que ya hace el prepare_search. Avisame que pensás.

Reply:
Bien, creo que viene por donde debería. Lo que si no entendí es lo del split, por lo que veo va a seguir funcionando si alguien escribe "Juan o Pepe" ¿No? Si es así, ninguna objeción por ese lado.

Comentario aparte y porque me acordé con esto, el otro día no me funcionó eso en el BestPractice, imagino que en la extracción anterior lo volamos. No es crítico que esté ahí, pero si en varias de las que vas a ir revisando.

Reply:
Claro, me "base" un poco en el de BestPractice y después me di cuenta que se perdía la posibilidad de meter operadores de todo tipo, habría que cambiar también ese...